### DISCUSSION TITLE
Implementation of Expander Graph Propagation

### DISCUSSION THREAD

**Discussion by hglee98** [pyg-team/pytorch_geometric#8039] [Q&A] [answered]
Hello! 
I'm curious that implementation of Expander Graph Propagation model in pytorch_geometric exists.
[https://arxiv.org/abs/2210.02997](url)

**Comment by rusty1s** [comment, marked as answer]
No, this model does not exist yet in PyG, sorry. Happy to take a contribution on this one.

**Reply by hglee98** [reply]
Thanks for your answer.
I'm currently reading the paper. So, I'd love to implement and contribute on it😊

**Reply by rusty1s** [reply]
Sounds cool :) Let me know if you wanna discuss anything.
